src/services/localSSHService.ts

```diff
@@ -83,12 +87,17 @@ export class LocalSSHService extends Disposable implements ILocalSSHService {
     }
 
     private async configureSettings({ proxyScript, launcher }: { proxyScript: string; launcher: string }) {
+        const gitpodHost = this.hostService.gitpodHost;
         const extIpcPort = Configuration.getLocalSshExtensionIpcPort();
-        const hostConfig = this.getHostSSHConfig(this.hostService.gitpodHost, launcher, proxyScript, extIpcPort);
-        await SSHConfiguration.ensureIncludeGitpodSSHConfig();
-        const gitpodConfig = await SSHConfiguration.loadGitpodSSHConfig();
-        gitpodConfig.addHostConfiguration(hostConfig);
-        await SSHConfiguration.saveGitpodSSHConfig(gitpodConfig);
+        const lockName = crypto.createHash('sha1').update(`${gitpodHost}_${extIpcPort}`).digest('hex').substring(0, 8);
```

Comment:
- No need for multiple locks as we are copying both files at the same time 
- Name should be constant, when updating gitpodHost or extIpcPort the event will fire in all opened window